qubes/ext/pci.py

```diff
@@ -150,16 +168,86 @@ def __init__(self, backend_domain, ident, libvirt_name=None):
                 raise UnsupportedDevice(libvirt_name)
             ident = '{bus}_{device}.{function}'.format(**dev_match.groupdict())
 
-        super().__init__(backend_domain, ident, None)
+        super().__init__(
+            backend_domain=backend_domain, ident=ident, devclass="pci")
+
+        if hasattr(self, 'regex'):
```

Comment:
This `if` was necessary when it was in a generic class. Here, the `regex` attribute is always set.

Reply:
ok